packages/angular_devkit/build_angular/src/utils/add-file-replacements.ts

```diff
@@ -22,53 +29,64 @@ export class MissingFileReplacementException extends BaseException {
   }
 }
 
-// Note: This method changes the file replacements in place.
-export function addFileReplacements(
-  root: Path,
-  host: virtualFs.AliasHost,
-  fileReplacements: FileReplacement[],
-): Observable<null> {
+export interface NormalizedFileReplacement {
+  replace: Path;
+  with: Path;
+}
 
+export function normalizeFileReplacements(
```

Comment:
Nit: file name does not match export name.

Reply:
Done.